apps/dashboard/app/javascript/packs/path_selector/data_table.js

```diff
@@ -104,15 +84,31 @@ jQuery(function () {
 
     $.fn.dataTable.ext.search.push(
         function (settings, data, dataIndex) {
-            if(table.getShowFiles()) {
-                return table.getShowDotFiles() || !data[1].startsWith('.');
+            let total = table.getShowDotFiles() + table.getShowFiles();
```

Comment:
These functions get called a bunch, is there some way to put the result in a variable given that they don't change?

Reply:
good call. you are right.